test/topology_experimental_raft/test_cdc_generation_data.py

```diff
@@ -0,0 +1,19 @@
+from test.pylib.manager_client import ManagerClient
+from test.pylib.rest_client import inject_error
+from test.topology.util import check_token_ring_and_group0_consistency
+
+import pytest
+
+@pytest.mark.asyncio
+async def test_send_data_in_parts(manager: ManagerClient):
```

Comment:
Please provide a Python docstring ("""blah blah blah""") describing how the test works (i.e.: it's a regression test, the injection forces the topology coordinator to send CDC generation data in multiple parts, if it didn't the command size would go over commitlog segment size limit making it impossible to commit and apply the command)

Reply:
Done.